EasyPost.Tests/PaymentMethodTest.cs

```diff
@@ -14,12 +14,41 @@ public void Initialize()
             _vcr = new TestUtils.VCR("payment_method", TestUtils.ApiKey.Production);
```

Comment:
This test file need to be changed to `BillingTest.cs`

Reply:
Resolved.